components/base/wheeled/wheeled_base.go

```diff
@@ -452,7 +452,9 @@ func (wb *wheeledBase) Close(ctx context.Context) error {
 	return wb.Stop(ctx, nil)
 }
 
-// Width returns the width of the base as configured by the user.
-func (wb *wheeledBase) Width(ctx context.Context) (int, error) {
-	return wb.widthMm, nil
+func (wb *wheeledBase) Properties(ctx context.Context, extra map[string]interface{}) (map[base.Feature]float64, error) {
+	return map[base.Feature]float64{
+		base.TurningRadiusM: 0.0,
+		base.WidthM:         float64(wb.widthMm) * 0.001, // convert to meters from mm
```

Comment:
Isn't this converting it from mm -> micro?

Reply:
no that's a division by 0.001, 600mm is .6m 
600mm is 600000um

Reply:
Damn. Brain not working. Listen, I don't do math, I just program

Reply:
It's fine, I'm here to be your opposite.